pkg/controllers/cluster/cluster_controller.go

```diff
@@ -531,8 +531,11 @@ func (c *Controller) tryUpdateClusterHealth(ctx context.Context, cluster *cluste
 		for _, clusterConditionType := range clusterConditionTypes {
 			currentCondition := meta.FindStatusCondition(cluster.Status.Conditions, clusterConditionType)
 			if currentCondition == nil {
-				klog.V(2).Infof("Condition %v of cluster %v was never updated by cluster-status-controller",
-					clusterConditionType, cluster.Name)
+				klog.V(2).InfoS(
+					"Condition was never updated by cluster-status-controller",
+					"condition", clusterConditionType,
+					"cluster", cluster.Name,
+				)
```

Comment:
```suggestion
				klog.V(2).InfoS("Condition was never updated by cluster-status-controller", "condition", clusterConditionType, "cluster", cluster.Name)
```
pernonally i would prefer to this style, but not an objection.